src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/validation/AnnotateVcfWithExpectedAlleleFraction.java

```diff
@@ -0,0 +1,107 @@
+package org.broadinstitute.hellbender.tools.walkers.mutect.validation;
+
+import htsjdk.variant.variantcontext.Genotype;
+import htsjdk.variant.variantcontext.VariantContext;
+import htsjdk.variant.variantcontext.VariantContextBuilder;
+import htsjdk.variant.variantcontext.writer.VariantContextWriter;
+import htsjdk.variant.vcf.VCFHeader;
+import htsjdk.variant.vcf.VCFHeaderLine;
+import htsjdk.variant.vcf.VCFHeaderLineType;
+import htsjdk.variant.vcf.VCFInfoHeaderLine;
+import org.apache.commons.math3.util.MathArrays;
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.cmdline.programgroups.VariantProgramGroup;
+import org.broadinstitute.hellbender.engine.FeatureContext;
+import org.broadinstitute.hellbender.engine.ReadsContext;
+import org.broadinstitute.hellbender.engine.ReferenceContext;
+import org.broadinstitute.hellbender.engine.VariantWalker;
+import org.broadinstitute.hellbender.utils.MathUtils;
+import org.broadinstitute.hellbender.utils.variant.GATKVariantContextUtils;
+
+import java.io.File;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+/**
+ * Given mixing weights of different samples in a pooled bam, annotate a corresponding
+ * vcf containing individual sample genotypes.  The formula is:
+ *
+ * Expected allele fraction = SUM_samples {mixing_fraction(sample) * [0 if hom ref, 0.5 is het, 1.0 if hom var]}
+ *
+ * Usage:
+ * java -jar gatk.jar AnnotateVcfWithExpectedAlleleFraction -V input.vcf -O output.vcf -mixingFractions mixingFractions.table
+ *
+ * Created by David Benjamin on 1/31/17.
+ */
+@CommandLineProgramProperties(
+        summary = "Annotate a vcf with expected allele fractions in pooled sequencing",
```

Comment:
Can you add a more detailed description.  Perhaps by copying some of the comments into the summary field?